.gitignore

```diff
@@ -0,0 +1,2 @@
+project/metals.sbt
+
```

Comment:
Here, I would add the `/.bloop` and `/.metals` directories as well.

Reply:
No worries, I have that in my global gitignore so I didn't see it. I also made a couple of more minor changes.

Reply:
@errikos This should be ready again.